iroh-blobs/src/store/bao_file.rs

```diff
@@ -9,34 +9,25 @@
 //! the file system for the data, outboard, and sizes file. There is also a
 //! combined implementation that starts in memory and switches to file when
 //! the memory limit is reached.
-use std::{
-    fs::{File, OpenOptions},
-    io,
-    ops::{Deref, DerefMut},
-    path::{Path, PathBuf},
-    sync::{Arc, RwLock, Weak},
-};
-
-use bao_tree::{
-    io::{
-        fsm::BaoContentItem,
-        outboard::PreOrderOutboard,
-        sync::{ReadAt, WriteAt},
-    },
-    BaoTree,
-};
+use std::fs::{File, OpenOptions};
+use std::io;
+use std::ops::{Deref, DerefMut};
+use std::path::{Path, PathBuf};
+use std::sync::{Arc, RwLock, Weak};
+
+use bao_tree::io::fsm::BaoContentItem;
+use bao_tree::io::outboard::PreOrderOutboard;
+use bao_tree::io::sync::{ReadAt, WriteAt};
+use bao_tree::BaoTree;
```

Comment:
do you know what ordering is going on here? it is not alphabetic

Reply:
It puts direct items always below sub-imports i think.

I don't really mind, as long as it's consistent in what it does :)

Reply:
I tried changing the order to some other stuff and it revert to whatever this is, so it seems to be consistent even if I don't fully get the order definition